Dfe.Data.SearchPrototype.Data/appsettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "AzureSearchServiceDetails": {
+    "serviceName": "s123d01-aisearch",
+    "indexName": "establishments"
+  },
```

Comment:
Personally I would include a placeholder for the API key here also (set to empty string, or something like `"populate me via secrets"`) for visibility.